EnvelopeItem/TransportConfiguration.php

```diff
@@ -99,24 +99,34 @@ public function setTimeToLive(int $timeToLive = null): self
         return $this;
     }
 
-    public function serialize()
+    public function serialize(): string
     {
-        return serialize(array(
-            'topic' => $this->topic,
-            'metadata' => $this->metadata,
-        ));
+        return serialize($this->__serialize());
     }
 
-    public function unserialize($serialized)
+    public function unserialize(string $serialized): void
```

Comment:
Probably we don't need ti since we removed the interface

Reply:
I left this method for supporting the previous version of the package if someone has been using this method in his code already

Reply:
This method is for magic call. No one uses it apart from PHP internally

Reply:
deleted